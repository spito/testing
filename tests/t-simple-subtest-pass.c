#include <cut.h>

TEST(withSubtest) {
    int i = 1;
    SUBTEST("S1") {
        ASSERT(i == 1);
    }
    ++i;
    SUBTEST("S2") {
        ASSERT(i == 2);
    }
    ++i;
    ASSERT(i == 3);
}
