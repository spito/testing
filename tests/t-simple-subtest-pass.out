[  1] withSubtest: 2 subtests
    S1.......................................................................OK
    S2.......................................................................OK
[  1] withSubtest (overall)..................................................OK

Summary:
  tests:          1
  succeeded:      1
  filtered out:   0
  suppressed:     0
  skipped:        0
  failed:         0
